openreview/venue_request/venue_request.py

```diff
@@ -349,17 +354,17 @@ def setup_submission_revision_stage(self):
                 ],
                 'default': 'Enable revision for all submissions',
                 'required': True,
-                'order': 37
+                'order': 38
             },
             'submission_revision_additional_options': {
-                'order': 38,
+                'order': 39,
                 'value-dict': {},
                 'description': 'Configure additional options in the revision. Valid JSON expected.'
             },
             'submission_revision_remove_options': {
-                'order': 39,
-                'values-dropdown': ['keywords', 'pdf', 'TL;DR'],
-                'description': 'Fields that should not be available during revision: keywords, pdf, TL;DR'
+                'order': 40,
+                'values-dropdown': ['title','authorids', 'abstract','keywords', 'pdf', 'TL;DR'],
```

Comment:
what about `authors`? should we add it to the list?

Reply:
Well the authors field is hidden in the submission form, right?

Reply:
The UI hides it but I'm not sure if the backend will try to validate it.

Reply:
Yep, I removed it